nitrocli/doc/nitrocli.1

```diff
@@ -35,6 +35,17 @@ printed information includes the SD card serial number, the encryption
 status, and the status of the volumes.
 
 .SS One-time passwords
+The Nitrokey Pro and the Nitrokey Store support the generation of one-time
+passwords using the HOTP algorithm according to RFC 4226 or the TOTP algorithm
+according to RFC 6238.
+The required data \- a name and the secret \- is stored on slots.
```

Comment:
Should that read "in slots"?

I read up a bit on hyphen usage in `*roff` and I don't think we are doing it correctly.
- `\-` is a minus sign and while that is valid for options (because a minus is the character used for them) [1] I don't think it should be used here.
- `-` is a hyphen

Now what we really want here (at least in my world view), is an em-dash, which is `\(em` [2].

In general, we probably want to change options like `--algorithm` to `\-\-algorithm`. What say you?

[1] https://liw.fi/manpages/ (ctrl-f "an unfor")
[2] http://man7.org/linux/man-pages/man7/man-pages.7.html (ctrl-f "Em-dash")